scraper_service/main.py

```diff
@@ -1,15 +1,20 @@
 import shutil
 from time import time
+from random import randint
```

Comment:
Can we remove this line as `randint ` is not being used?

Reply:
yes, let's remove it.